eZ/Bundle/EzPublishCoreBundle/EventListener/BlockCacheResponseListener.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\EventListener;
+
+use eZ\Publish\Core\MVC\Symfony\View\CachableView;
+use eZ\Publish\Core\MVC\Symfony\View\BlockView;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Configures the Response cache properties.
+ */
+class BlockCacheResponseListener implements EventSubscriberInterface
+{
+    /**
+     * True if view cache is enabled, false if it is not.
+     *
+     * @var bool
+     */
+    private $enableViewCache;
+
+    public function setEnableViewCache($enableViewCache)
+    {
+        $this->enableViewCache = $enableViewCache;
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [KernelEvents::RESPONSE => ['configureBlockCache', -10]];
+    }
+
+    public function configureBlockCache(FilterResponseEvent $event)
+    {
+        $view = $event->getRequest()->attributes->get('view');
+        if (!$view instanceof CachableView || !$view instanceof BlockView) {
+            return;
+        }
+
+        if (!$this->enableViewCache || !$view->isCacheEnabled()) {
```

Comment:
Should be a `&&` to stick with global view cache setting (like https://github.com/ezsystems/ezpublish-kernel/pull/1472)